projects/canopy/src/lib/card/card.interface.ts

```diff
@@ -1,2 +1,16 @@
 export const lgCardToggleIdPrefix = 'lg-card-toggle-';
 export const lgCardPanelIdPrefix = 'lg-card-panel-';
+export type CardVariant = 'default' | 'navigation' | 'promotion';
+
+export type Orientation =
+  | 'horizontal'
+  | 'vertical'
+  | 'reverse-horizontal'
+  | 'reverse-vertical';
+export interface OrientationResponsive {
```

Comment:
Could you please add a new line before the interface?

Reply:
Done and moved :)